ddtrace/debugging/_debugger.py

```diff
@@ -611,7 +593,7 @@ def _probe_wrapping_hook(self, module: ModuleType) -> None:
             else:
                 context = DebuggerWrappingContext(
                     function,
-                    collector=self._collector,
+                    collector=self.__uploader__.get_collector(),
```

Comment:
maybe we can hide collector / merge it with __uploader__? 

most calls here are just self.__uploader__.get_collector() and they all just use push(signal)

maybe just add: self.__uploader__.push(signal)

Reply:
Yes I was thinking about merging the collector's API with that of the uploader, but I thought it might have been "cleaner" to keep them separate. Perhaps we can revisit this in a follow-up?